draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1347,56 +1347,79 @@ Content-Type: text/plain
-
-<t>Note that names can be registered by third parties (including the
-Expert(s)), if the Expert(s) determines that an unregistered name is widely
-deployed and not likely to be registered in a timely manner otherwise.</t>
-
+<t>
+   The "Hypertext Transfer Protocol (HTTP) Field Name Registry" defines the
+   namespace for HTTP field names.
+</t>
+<t>
+   Any party can request registration of a HTTP field. See <xref
+   target="considerations.for.new.header.fields"/> for considerations to take
+   into account when creating a new HTTP field.
+</t>
+<t>
+   The "Hypertext Transfer Protocol (HTTP) Field Name Registry" is located at
+   <eref target="https://www.iana.org/assignments/http-fields/"/>.
+   Registration requests can be made by following the instructions located
+   there or by sending an email to the "ietf-http-wg@ietf.org" mailing list.
+</t>
+<t>
+   Field names are registered on the advice of a Designated Expert
+   (appointed by the IESG or their delegate). Fields with the status
+   'permanent' are Specification Required
+   (<xref target="RFC8126" x:sec="4.6" x:fmt=","/>).
+</t>
+<t>
+   Registration requests consist of at least the following information:
+</t>
+<dl newline="true">
+  <dt>Field name:</dt>
+  <dd>
+    The requested field name. It &MUST; conform to the
+    field-name syntax defined in <xref target="field.names"/>, and &SHOULD; be
+    restricted to just letters, digits, hyphen ('-') and underscore ('_')
+    characters, with the first character being a letter.
+  </dd>
+  <dt>Status:</dt>
+  <dd>
+    "permanent" or "provisional".
+  </dd>
+  <dt>Specification document(s):</dt>
+  <dd>
+    Reference to the document that specifies
+    the field, preferably including a URI that can be used to retrieve
+    a copy of the document. An indication of the relevant section(s) can also
+    be included, but is not required.
+  </dd>
+</dl>
+<t>
+   And, optionally:
+</t>
+<dl>
+  <dt>Comments:</dt>
+  <dd>
+    Additional information, such as about reserved entries.
+  </dd>
+</dl>
+<t>
+   The Expert(s) can define additional fields to be collected in the
+   registry, in consultation with the community.
+</t>
+<t>
+   Standards-defined names have a status of "permanent". Other names can also
+   be registered as permanent, if the Expert(s) find that they are in use, in
+   consultation with the community. Other names should be registered as
+   "provisional".
+</t>
+<t>
+   Provisional entries can be removed by the Expert(s) if &mdash; in consultation
+   with the community &mdash; the Expert(s) find that they are not in use. The
+   Experts can change a provisional entry's status to permanent at any time.
+</t>
+<t>
+   Note that names can be registered by third parties (including the
+   Expert(s)), if the Expert(s) determines that an unregistered name is widely
+   deployed and not likely to be registered in a timely manner otherwise.
+</t>
```

Comment:
This should really be separate :)

Reply:
separate from what? (there's no change here to what we have right now)